utils/vulcan.py

```diff
@@ -4,8 +4,12 @@
 from utils.modules_ext import *
 from utils.helper import *
 
-# run VULCAN/atmosphere chemistry
 def RunVULCAN( atm, time_dict, loop_counter, dirs, runtime_helpfile, COUPLER_options ):
```

Comment:
If a different functionality is used for VULCAN, delete this function?

Reply:
The current functionality for VULCAN is the offline chemistry stuff. There's no way to run it 'online' at the moment, although this should probably be addressed by issue #6 in the future.